firefox-ios/Client/Frontend/Browser/BrowserViewController/Extensions/BrowserViewController+URLBarDelegate.swift

```diff
@@ -326,6 +327,17 @@ extension BrowserViewController: URLBarDelegate {
         )
         searchController?.searchQuery = text
         searchLoader?.query = text
+        searchController?.interactionType = determineInteractionType(for: text)
+    }
+
+    func determineInteractionType(for text: String) -> TelemetryWrapper.EventExtraKey.UrlbarTelemetry.Interaction {
+        guard let savedQuerry = searchController?.savedQuery else { return .typed }
+        if text.count - savedQuerry.count == 1 {
```

Comment:
nit: since we used this twice, maybe create a constant that contains the value and use it in the two places?

Reply:
this is only useful as a response to the search string getting edited, the interactionType is set in different places based on the perceived interaction. By moving it I got rid of the parameters and it looks cleaner now